src/backend/MoveList.js

```diff
@@ -28,37 +28,37 @@ export function MoveList(props){
 
 	return (<div>
 				<div style ={{paddingBottom: "1em", paddingTop: "1em"}}>
-					<Button type="button" className="btn btn-lg btn-light" style = {{width:"20em"}} onClick={all_moves["Take General Income"](props.roomName, props.currentTurn, props.playerName, props.activePlayerID)}
+					<Button type="button" className="btn btn-lg btn-light" style = {{width:"20em"}} onClick={all_moves["Take General Income"](props.roomName, props.currentTurn, props.playerName, props.activePlayerID, null)}
```

Comment:
@aravind-segu didn't read all the code, but if we pass null wouldn't we get an error when the function is called. Or is it guaranteed not to be called for this move type?